### PR TITLE
docs: sd-accordion-group docs update [skip chromatic]

### PR DESCRIPTION
**Pull request by smfonseca**
## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

**Review by Vahid1919 — APPROVED** [review]

**Comment by mariohamann** [comment]
Sorry, I forgot to push my changes I had discussed some days ago...

CleanShot 2024-08-09 at 08 52 32@2x

@coraliefeil I think you already updated, but it now looks like this:

CleanShot 2024-08-09 at 08 52 52@2x

File: packages/components/src/components/accordion-group/accordion-group.stories.ts [hunk 1]
```diff
@@ -13,6 +13,10 @@ const { generateTemplate } = storybookTemplate('sd-accordion-group');
  *  **Related templates**:
```
Comment (coraliefeil) [inline packages/components/src/components/accordion-group/accordion-group.stories.ts:13]: sd-accordion-group

Used to group multiple accordions together.

Comment (smfonseca) [reply packages/components/src/components/accordion-group/accordion-group.stories.ts:13]: Updated

**Comment by coraliefeil** [comment]
All good. THX

**Pull request merged**